daemons/based/based_callbacks.c

```diff
@@ -1430,7 +1430,9 @@ cib_process_command(xmlNode * request, xmlNode ** reply, xmlNode ** cib_diff, gb
         cib_replace_notify(origin, the_cib, rc, *cib_diff, change_section);
     }
 
-    pcmk__xml_log_patchset(LOG_TRACE, *cib_diff);
+    pcmk__output_set_log_level(logger_out, LOG_TRACE);
+    logger_out->message(logger_out, "xml-patchset", *cib_diff);
```

Comment:
Where we're using this approach, do we need to reset the log level to whatever it originally was, or do all the uses already explicitly set the desired level?

Reply:
All three of the uses explicitly set the desired level. It seems much clearer and less error-prone that way, unless we need something more complex later where we set the log level for a whole block and something called within the block also needs to set the log level.